packages/edit-site/src/components/navigation-sidebar/navigation-panel/style.scss

```diff
@@ -1,7 +1,6 @@
 .edit-site-navigation-panel {
 	animation: edit-site-navigation-panel__slide-in 0.1s linear;
 	height: 100%;
```

Comment:
(Not caused by this PR, but it's more a mental note to address it at some point)

The panel is 100% height, but in fact it should be slightly shorter: it's pushed down by the header (60px + 1px of border), and it's supposed to end above the footer (25px, only on "> mobile").

I guess we could remove those 86px from the height with `calc()` or as a bottom padding.

Somewhat related: the Navigation component hides both overflows, while it only really needs to hide `overflow-x`.
Hiding `overflow-y` means that if the menu is taller than the screen, it gets cut off.